src/public/interfaces.ts

```diff
@@ -158,6 +158,15 @@ export interface LocaleInfo {
   longTime: string;
 }
 
+/**
+ * Allowed user file open preferences
+ */
+export enum FileOpenPreference {
+  Teams = 'teams',
```

Comment:
Any chance we can make this a little more generic - e.g. Hub, InApp, or something like that? We're soon going to be replacing the Teams Client SDK with the MetaOS App SDK which will used to host apps in a number of different Office hubs like Outlook, Office.com, etc.

Reply:
Yes, we should be able to. But we will need to write a mapping function in teams to read&pass this values as inApp.

Reply:
Have updated to 'inApp'